doc/fluid/design/dist_train/large_model.md

```diff
@@ -0,0 +1,44 @@
+# Design Doc: Large Model
+
+## Abstract
```

Comment:
Need to tell about the background, why we need this feature.

Reply:
Done.